### ISSUE TITLE
SUMMARY: Meets Expectations

### ISSUE THREAD

**Issue by sdhunt** [aigarsgaa/colmar-academy#5]
_Grade:_ __Meets Expectations__

Overall, a solid piece of work -- great job!

* Your source code is clean, well-structured, and easily readable
* Your page renders as per the spec, and handles window resizing smoothly
* You made good use of HTML semantic elements, and correctly used HTML entities

To challenge yourself further, you might wish to look at making the "boxes" for the items "Orientation Date", "Our Campus", and "Our Guest Lecturer" clickable (not just the text link), and include the `:hover` and `transition`  treatment to those, to show a visual change when the user hovers the mouse. (Tip: use `cursor: pointer;`  attribute to change the mouse to "pointing hand" icon). 

Same can be suggested for the Course boxes in the "Start Learning" section. 

In summary: great job, but some scope for making it even better.

I hope you find the following article interesting:
* https://learn.shayhowe.com/html-css/writing-your-best-code/

Keep up the good work!
